src/ocean/LibVersion.d

```diff
@@ -0,0 +1,9 @@
+module ocean.LibVersion;
+
+const has_features_2_6 = true;
```

Comment:
Yes, that is the plan. If design looks OK, I'll make separate pull requests per each supported branch to define consts separately.